README.md

```diff
@@ -1,3 +1,19 @@
+<!-- PROJECT ICON -->
+
+<p align="center">
+    <a href="https://developer.ibm.com/languages/java/semeru-runtimes/">
+        <img src="./SemeruRuntimes.svg" alt="Semeru" width="200">
+    </a>
+</p>
+<br />
+
+[![Download latest](https://img.shields.io/github/downloads/ibmruntimes/semeru11-certified-binaries/latest/total)](https://github.com/ibmruntimes/semeru11-certified-binaries/tags)
```

Comment:
Is "latest" redundant if we are keeping this list current?

Reply:
Yes, the latest is the same as jdk-11.0.17+8_openj9-0.35.0 at the moment. The latest will always point to the new release at the time.

Reply:
So, do we need/want both?

Reply:
I think it can be helpful in the future when the download stats for the new release is not added in time.